eng/dockerrun.sh

```diff
@@ -148,6 +148,8 @@ docker run $INTERACTIVE -t --rm --sig-proxy=true \
     -e BUILD_SOURCEBRANCH \
     -e BUILD_BUILDNUMBER \
     -e BUILD_SOURCEVERSION \
+    -e SYSTEM_TEAMPROJECT \
+    -e DOTNECLIMSRC_READ_SAS_TOKEN \
```

Comment:
There are a few more variables that need to be set in this invocation. See 
https://dev.azure.com/dnceng/internal/_git/dotnet-core-sdk?path=%2Feng%2Fdockerrun.sh&version=GBinternal%2Friarenas%2Ffix-docker&_a=contents&line=153&lineStyle=plain&lineEnd=157&lineStartColumn=1&lineEndColumn=1

Reply:
Yeah, got it. I'll address that.

Reply:
Done

Reply:
This LGTM now.